Artifacts/MsSql/Data/Security/1030-DataStandard3.3a-ResourceClaimMetadata.sql

```diff
@@ -55,3 +55,15 @@
     (SELECT claimSetId  FROM dbo.ClaimSets
     WHERE ClaimSetName IN ('SIS Vendor','Ed-Fi Sandbox','District Hosted SIS Vendor')) AS cs
     WHERE ResourceName = 'organizationDepartment';
+
+	INSERT INTO [dbo].[ClaimSetResourceClaimActionAuthorizationStrategyOverrides]
+		([AuthorizationStrategyId]
+		,[ClaimSetResourceClaimActionId])
+	SELECT @noFurtherAuthRequiredAuthorizationStrategyId, crc.ClaimSetResourceClaimActionId
+	FROM [dbo].[ClaimSetResourceClaimActions] crc
+	INNER JOIN [dbo].[Actions] a ON
+		crc.ActionId=a.Actionid and 
+	    a.ActionName IN ('Create','Read','Update','Delete')
+	INNER JOIN [dbo].[ResourceClaims] r ON 
```

Comment:
Inner join is missing dbo.ClaimSets
    WHERE ClaimSetName IN ('SIS Vendor','Ed-Fi Sandbox','District Hosted SIS Vendor')